CMS-Enterprise/mint-app
# [MINT-3099][MINT-3100]/cr tdl echimp data

# MINT-3099, [MINT-3100](https://jiraent.cms.gov/browse/MINT-3100)

## Description

1. Introduce new echimp CR and TDL data type
2. Introduce new seed bucket for local CR / TDL data (more to come later)
   a. Note that there is not a model plan associated with the seeded data currently, that is a more involved effort
4. Rudimentary cache with expiration
5. parsing of flags into bools 
6. Postman collection updates


## How to test this change

1.  Seed the database
2. Use the postman collection to view available data.
3. Ensure \N is removed from text. 
4. Ensure that html is removed from all text.
5. Ensure that the postman collection provides the data in a format needed for the frontend
    a. note that there is not yet seeded model plans corresponding to the test data, as it involves reworking the seeding process.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
